parser/parser.go

```diff
@@ -160,9 +175,40 @@ func isGit(u *url.URL) bool {
 	if strings.ToLower(u.Scheme) == iterator.GitSchemeRaw {
 		return true
 	}
-	if strings.ToLower(u.Scheme) == iterator.HttpsSchemeRaw && strings.ToLower(u.Host) == "github.com" {
+	if strings.ToLower(u.Scheme) == iterator.HttpsSchemeRaw && (strings.ToLower(u.Host) == "github.com" || strings.ToLower(u.Host) == "webrtc.googlesource.com") {
 		return true
 	}
 
 	return false
 }
+
+// isZip is a helper method to determine whether an url has a Zip extension.
```

Comment:
rephrase

Reply:
Fixed. Rephrased to "isZip is a helper method to determine whether a string has a Zip extension suffix"